packages/cli/templates/default-plugin/tsconfig.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "@backstage/cli/config/tsconfig.json",
+  "include": [
+      "src",
+      "dev",
+      "migrations"
+  ],
+  "exclude": ["node_modules"],
+  "compilerOptions": {
+    "outDir": "dist-types",
```

Comment:
Best set `"rootDir": "."` too, just in case

Reply:
ok